hack/make-rules/verify.mk

```diff
@@ -0,0 +1,29 @@
+GO_VERSION:=1.13
+CODE_MAINT += go-version
+.PHONY: go-version
+go-version:
+	@(go version | grep -q 'go$(GO_VERSION)\(\.[0-9]*\)\? ') || \
+	echo 'WARNING: bad go version to fix run: eval "$$(gimme $(GO_VERSION))"'
```

Comment:
I would omit this and just declare in the README that go 1.12 or later is required for devs

Reply:
OK will do.